tests/docker/ducker-ak

```diff
@@ -311,15 +313,16 @@ use only ${num_nodes}."
 
     ducker_build "${image_name}"
 
-    docker inspect --format='{{.Config.Labels}}' --type=image "${image_name}" | grep -q 'ducker.type'
-    local docker_status=${PIPESTATUS[0]}
-    local grep_status=${PIPESTATUS[1]}
+    docker inspect --format='{{.Config.Labels}}' --type=image "${image_name}" | grep -q 'ducker.creator'
```

Comment:
Why change the argument to grep?

Reply:
`ducker.creator` is the label used in the [Dockerfile](https://github.com/apache/kafka/blob/db688b1a5e5904eed5849fbab4940585e5a3d646/tests/docker/Dockerfile#L32).